lib/config/index.ts

```diff
@@ -1,3 +1,4 @@
+import fs from 'fs';
```

Comment:
```suggestion
import fs from '../util/fs';
```
you need to use our fs wrapper